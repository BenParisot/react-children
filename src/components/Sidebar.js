import React from 'react';
import PropTypes from 'prop-types';
import Styles from './main.css';

export default function Sidebar({ children }) {
  return (
    <nav className={Styles.sidebar}>
      {children}
    </nav>
  );
}

Sidebar.propTypes = {
  children: PropTypes.node
};
