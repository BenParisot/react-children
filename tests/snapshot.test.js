import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from '../src/components/Sidebar';

describe('Sidebar component', () => {
  it('creates a sidebar', () => {
    const wrapper = shallow(<Sidebar />);
    expect(wrapper).toMatchSnapshot();
  });
});
