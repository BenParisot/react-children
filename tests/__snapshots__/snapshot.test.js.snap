// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar component creates a sidebar 1`] = `
<nav
  className="sidebar"
/>
`;
